operations/network/customer_accessible_networks/can_to_chn/network/network_upgrade_1.2_to_1.3.md

```diff
@@ -96,13 +97,13 @@ For *each* switch in the system, determine the configuration differences.
 - (`ncn-m001#`) **Only** during 1.2 to 1.3 upgrades.
 
    ```bash
-   canu test network --csm 1.2
+   canu test --csm 1.2
```

Comment:
This same error also exists on this page: [operations/network/management_network/1.2_to_1.3_upgrade.md](https://github.com/Cray-HPE/docs-csm/blob/b616bb7f9607bc9f770a14f238ee79721cff7373/operations/network/management_network/1.2_to_1.3_upgrade.md?plain=1#L201)